docker-compose.yml

```diff
@@ -45,7 +47,7 @@ services:
     networks:
       - broker
     working_dir: '/home/app'
-    entrypoint: bash -c 'npm run start-sparkswapd'
+    entrypoint: bash -c 'npm run start-sparkswapd-watch'
```

Comment:
@treygriffith good catch. This can actually go in the `regtest` sample docker-compose file. We need this line so that we can watch the file system for changes (hot reloading for pm2)